6407-determine if Liberty is using a specific feature (#6470)
OpenLiberty/docs: modules/reference/pages/feature/feature-overview.adoc

* The feature:microProfile[display=MicroProfile] convenience feature enables all the Open Liberty features that support the MicroProfile platform.
* The feature:enterpriseBeans[display=Jakarta Enterprise Beans] convenience feature automatically enables all Open Liberty features that support enterprise beans functions.

> Occasionally, Liberty employs functions that are not listed in the `server.xml` file. To confirm if a particular feature is enabled in a Liberty server, you must inspect the `CWWKF0012I` message in the `console.log`, `messages.log`, or trace log from the Liberty server. This message provides a comprehensive list of all the features that are installed and running on the Liberty server.

Reviewer: ```suggestion
Occasionally, Liberty features that are not listed in the `server.xml` file are enabled automatically by other features. To confirm whether a particular feature is enabled in a Liberty server, inspect the `CWWKF0012I` message in the `console.log`, `messages.log`, or trace log from the Liberty server. This message provides a comprehensive list of all the features that are installed and running on the Liberty server.
```
